core/src/main/scala/chisel3/experimental/package.scala

```diff
@@ -166,9 +166,4 @@ package object experimental {
   val prefix = chisel3.internal.prefix
   // Use to remove prefixes not in provided scope
   val noPrefix = chisel3.internal.noPrefix
-
-  /** Base simulation-only component. */
-  abstract class BaseSim extends NamedComponent {
```

Comment:
is a printf, assume, etc still a `NamedComponent`?

Reply:
Yes. I just renamed the base class to `VerificationStatement` and moved it into the `chisel3` package.